core/api/v1/keytransparency_proto/keytransparency.proto

```diff
@@ -28,7 +28,7 @@ import "crypto/keyspb/keyspb.proto";
 import "crypto/sigpb/sigpb.proto";
 import "trillian.proto";
 import "trillian_map_api.proto";
-import "api/v1/keytransparency_proto/admin.proto";
+import "v1/keytransparency_proto/admin.proto";
```

Comment:
Is the removal of 'api/' here a consequence of moving the gen.go file into the api/ dir?

Reply:
That's right. Due to the `-I=.` line in the protoc invocation, the root search path for imports will be `core/api`.